pretyping/tacred.ml

```diff
@@ -1182,6 +1183,13 @@ let unfold env sigma name c =
  * at the occurrences of occ_list. If occ_list is empty, unfold all occurrences.
  * Performs a betaiota reduction after unfolding. *)
 let unfoldoccs env sigma (occs,name) c =
+  begin match name with
+    | EvalVarRef id -> begin try ignore (lookup_named id env) with Not_found as exn ->
+        let _, info = Exninfo.capture exn in
```

Comment:
I don't think there's any point in capturing here.

Reply:
I agree, and the layout is not very idiomatic, you should probably add a line return as in `EvalVarRef id ->\n`.

Reply:
The capturing pointless, because the error location will point to `unfold` tactic anyway, right?